sirepo/job_driver/sbatch.py

```diff
@@ -33,14 +34,27 @@ class SbatchDriver(job_driver.DriverBase):
     __instances = PKDict()
 
     def __init__(self, op):
+        def _op_queue_size(op_kind):
+            return self.cfg.num_concurrent_op_run if op_kind == job.OP_RUN else 1
+
         super().__init__(op)
         self.pkupdate(
             # before it is overwritten by prepare_send
             _local_user_dir=pkio.py_path(op.msg.userDir),
             _srdb_root=None,
-            # we allow one of each op type in. This is essentially a no-op (ha ha)
-            # but makes it easier to code the other cases.
-            cpu_slot_q=job_supervisor.SlotQueue(len(job_driver.OPS_THAT_NEED_SLOTS)),
+            # Allow num_concurrent_op_run and every other op type to
+            # run (assume OP_RUN is one of OPS_THAT_NEED_SLOTS). This
+            # is essentially a no-op (sbatch constrains its own cpu
+            # resources) but makes it easier to code the other cases.
+            cpu_slot_q=sirepo.job_supervisor.SlotQueue(
+                len(job_driver.OPS_THAT_NEED_SLOTS)
+                + self.cfg.num_concurrent_op_run
+                - 1,
```

Comment:
`-1` is unclear. Is this subtracting RUN from OPS_THAN_NEED_SLOTS? Document that, e.g. `-1 accounts for run having num_concurrent_op_run and not needing a separate slot`. I'm not sure about the coupling here. Do we even run analysis?

Reply:
My hope was that the comment above served the purpose. Maybe I should move it above the -1?